site/bps/BP-25-MovingChecksumToProto.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "BP-25: MovingChecksumToProto--Refactor the checksum part of bookkeeper"
+issue: https://github.com/apache/bookkeeper/issues/1007
+state: "Under Discussion"
```

Comment:
@ArvinDevel can you update this to "Accepted"? so I can merge this BP proposal.

Reply:
updated